portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java

```diff
@@ -1794,6 +1779,50 @@ private boolean _processCompanyInactiveRequest(
 		return true;
 	}
 
+	private void _verifyFileEntry(
```

Comment:
Can you rename this method to `_check*`? "Verify" has a very specific meaning that is not related to this usage.